packages/sdk-v2/src/sdk/claim/sdk.ts

```diff
@@ -42,7 +43,7 @@ export class ClaimSDK extends BaseSDK {
       return fvSig;
     };
 
-    const getLink = (
+    const getLink = async (
```

Comment:
I do't like you change existing api by making one of the key functions async
do we really have to wait until fvSig set ? could we just do safeSet() without awaiting ?

Reply:
See comment thread above and this commit with the requested changes: https://github.com/GoodDollar/GoodWeb3-Mono/pull/130/commits/3a197ac5090b24af7a7c7585a377b338b4567433